md2conf/converter.py

```diff
@@ -305,6 +369,17 @@ def sanitize_confluence(html: str) -> str:
     return _content_to_string(root)
 
 
+def _content_title(root: ET.Element) -> str:
+    xml = ET.tostring(root, encoding="utf8", method="xml").decode("utf8")
+    m = re.match(r".*<h1>(.*)</h1>.*", xml, re.DOTALL)
+
+    # if the contents of the page are empty
+    if m == None:
```

Comment:
While it's somewhat of a personal taste question, I prefer to use `m is None` since `None` is a unique object in Python.